### PR TITLE
fix: update XML files to address parsing error

### PR DESCRIPTION
**Pull request by bjagg**
Several institutions and myself experienced issues with SCP seeing the same XML parsing error. I updated most of the XML files to match Spring version, and XML standards.

### CODE REVIEW

File: src/main/resources/context/applicationContext.xml [hunk 1]
```diff
@@ -24,9 +25,9 @@
     xmlns:p="http://www.springframework.org/schema/p"
     xmlns:tx="http://www.springframework.org/schema/tx"
     xsi:schemaLocation="http://www.springframework.org/schema/beans
-    http://www.springframework.org/schema/beans/spring-beans-2.5.xsd
-    http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context-2.5.xsd
-    http://www.springframework.org/schema/tx http://www.springframework.org/schema/tx/spring-tx.xsd">
+    https://www.springframework.org/schema/beans/spring-beans.xsd
+    http://www.springframework.org/schema/context https://www.springframework.org/schema/context/spring-context.xsd
```
Comment (ChristianMurphy) [inline src/main/resources/context/applicationContext.xml:29]: Could the other urls also move to https?

**Review by ChristianMurphy — APPROVED** [review]
I haven't tested this yet.
Spot checking the XML and looking at CI this seems fine.

File: src/main/resources/context/applicationContext.xml (hunk 1 above)
Comment (bjagg) [reply src/main/resources/context/applicationContext.xml:29]: The first of the pair are actually identifiers. The second ones are actual URLs.

**Comment by robin850** [comment]
Just chiming in to say that we've applied this fix on our production servers and this is working fine. Thanks for the quick fix ; the error trace was too vague.

**Review by cbeach47 — APPROVED** [review]
Nice 👍

**Pull request merged**